src/main/scala/viper/silver/testing/SilSuite.scala

```diff
@@ -22,6 +22,8 @@ abstract class SilSuite extends AnnotationBasedTestSuite with BeforeAndAfterAllC
     */
   def verifiers: Seq[Verifier]
 
+  def verifiers(configMap : Map[String, Any]) : Seq[Verifier] = verifiers
```

Comment:
```suggestion
  def verifiers(configMap : Map[String, Any]): Seq[Verifier] = verifiers
```
(fewer spaces around `:`)